issue #81 fixed

AB000001 opened the pull request:
## Description
Fixes #81. Converted gameControllers.js to gameControllers.ts

## Motivation and Context
Issue #81 fixed

kuv2707 commented on .gitignore:
```diff
@@ -1,4 +1,5 @@
-.env
\ No newline at end of file
+.env
+*.json
```
What prompted this change?

AB000001 replied:
I had changed it for my local purpose. We can ignore these entries from. Gitignore

kuv2707 reviewed (commented):
Great work!
Please take a moment to read the contributing guidelines (in the file contributing.md) and format the PR description and commit message accordingly.

The PR description should have the clause `Fixes #issueno`.

Once that is done, this can be merged

kuv2707 commented:
@AB000001 Still on it?

AB000001 commented:
> Great work! Please take a moment to read the contributing guidelines (in the file contributing.md) and format the PR description and commit message accordingly.
> 
> The PR description should have the clause `Fixes #issueno`.
> 
> Once that is done, this can be merged

Modified the Pull request description to include issue #81. and modified the commit remark also

kuv2707 commented:
@AB000001 The commits don't seem to follow the guidelines. 
There should only be one commit with a properly formatted message. The contributing docs can help you with it. 
You need to "squash" all the commits into one.

Also I'd expect only one file change as per the issue description, but here there are two.

kuv2707 commented:
It seems like this issue has been addressed in a recent commit, since the file is now in typescript.
In that case, it would be useless to work further on it.

You can still use this PR to learn some git commands, and I'll approve it once it is in good shape, so that it is counted as a contribution. (Just be sure not to "rebase" this branch).

I'll unassign you from the issue so that you can take up others.
Thanks for all the work!

kuv2707 commented:
closing due to inactivity
